### DISCUSSION TITLE
Can you provide a model based on tensorflow lite? How much can you provide tflite model ？

### DISCUSSION THREAD

**Discussion by empty123321** [snakers4/silero-vad#559] [Q&A] [answered]
Can you provide a model based on tensorflow lite?

**Comment by snakers4** [comment, marked as answer]
> tensorflow lite?

Hi,

Such conversions are typically difficult and require model re-training because of framework limitations and sometimes end users have hardware limitations too. 

Typically we do such projects for companies for a fee.

**Reply by empty123321** [reply]
How much？
